CHANGELOG.md

```diff
@@ -4,12 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/d4l-data4life/hc-sdk-kmp/compare/1.7.0...main)
+## [Unreleased](https://github.com/d4l-data4life/hc-sdk-kmp/compare/1.7.3...main)
 ### Added
-- SDKContract method to get active user session token
 ### Changed
-- User service getSessionToken method now returns with an active token not the last known.
 ### Removed
 ### Fixed
 ### Updated
 ### Migration
+
+## [1.7.3](https://github.com/d4l-data4life/hc-sdk-kmp/compare/1.7.0...1.7.3)
```

Comment:
was the previous version the 1.7.0 or 1.7.1 or 1.7.2

Reply:
The previous version was the v1.7.2, but it seems i was just a bump without actual changes. I updated the compare url to `v1.7.2..v1.7.3`